src/main/scala/ironoxide/v1/common/UserOrGroupId.scala

```diff
@@ -3,23 +3,26 @@ package ironoxide.v1
 import cats.effect.Sync
 import com.ironcorelabs.{sdk => jsdk}
 
-/** ID of a user or a group. */
-sealed trait UserOrGroupId extends Product with Serializable {
-  def id: String
-}
+package common {
+
+  /** ID of a user or a group. */
+  sealed trait UserOrGroupId extends Product with Serializable {
+    def id: String
+  }
 
-object UserOrGroupId {
-  def apply(uid: jsdk.UserOrGroupId): UserOrGroupId =
-    if (uid.isUser)
-      ironoxide.v1.user.UserId(uid.getId)
-    else
-      ironoxide.v1.group.GroupId(uid.getId)
+  object UserOrGroupId {
+    def apply(uid: jsdk.UserOrGroupId): UserOrGroupId =
+      if (uid.isUser)
+        ironoxide.v1.user.UserId(uid.getId)
+      else
+        ironoxide.v1.group.GroupId(uid.getId)
+  }
 }
 
 package user {
```

Comment:
I would put UserId and GroupId in common as they are pretty basic types that are used in a variety of places.

Reply:
I don't really think the name `common` is meant to indicate that the types are widely used, but I made the change anyway because those types were in a confusing disagreement between file and package.